src/routes.json

```diff
@@ -42,44 +42,44 @@
     {
         "name": "conference-index",
         "pattern": "^/conference/?$",
-        "routeAlias": "/conference(?!/201[4-5])",
+        "routeAlias": "/conference(?!/201[4-7])",
```

Comment:
Ahhhh, I think I misunderstood the routesAlias file and what exactly it means to do. Thanks for clarifying this! I'll check out the webpack thing and see if I can solve this issue :)